onboard/src/components/Accordion.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import Typography from "@mui/material/Typography";
+import Grid from "@mui/material/Grid";
+import { Divider } from "@mui/material";
+import Box from "@mui/material/Box";
+import { Button } from "@mui/material";
+import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
+import ArrowDropUpIcon from "@mui/icons-material/ArrowDropUp";
+
+// Flexible Accordion module with a number of props, some of which are optional (importance, open)
+function Accordion({ title, open, index, content }) {
+  const [isActive, setIsActive] = useState(open ? true : false);
+
+  return (
+    <Grid item xs={12} sx={{ marginTop: 7 }} key={index + Math.random()}>
+      <Divider variant="fullWidth" light className="width100Percent" />
```

Comment:
`.width100Percent {
  width: 100%;
}`

Forgot this class. May be good to add in an additional css imported file, otherwise delete the class.

Reply:
I'm working on this.